python/cudf/cudf/tests/test_list.py

```diff
@@ -107,6 +109,27 @@ def test_listdtype_hash():
     assert hash(a) != hash(c)
 
 
+@settings(deadline=None)
+@given(
+    st.builds(
+        lambda x: [x],
+        st.recursive(
+            st.integers(min_value=-100, max_value=100),
```

Comment:
Worth testing other types? (float, datetime, timedelta?)

Reply:
Done as part of reworking the test.